index.html

```diff
@@ -0,0 +1,117 @@
+              <a class="nav-link" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Sponsor</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">News</a>
+            </li>
+            <li class="nav-item emphasis">
+              <a class="nav-link orange" href="#">Campaign</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    
+    
+    <section class="home-section">
+<img id="toggle_menu" class="hamburger" src="./img/menu.png" alt="Hamburger Menu">   
+<div id="menu" class="main-nav">
+    
+        <ul id="menu-ul" class="d-flex main-nav-ul">
+        <li id="close_menu" class="close_menu">x</li>
+        <li class="top-nav-list"><a href="/about.html">About</a></li>
+        <li class="top-nav-list currunt-tab"><a href="/index.html">Program</a></li>
+        <li class="top-nav-list">Join</li>
+        <li class="top-nav-list">Sponsor</li>
+        <li class="top-nav-list">News</li>
+       </ul>
+    </div>
+<div class="home-contanier">
+           <h1 class="home-title">Space Tech Expo 2021</h1>
+
+           <h2 class="home-haeder">SPACE TECH EXPO EUROPE RETURNS FOR 2021</h2>
+
+           <p class="home-p">
+            Register now for Europe’s largest exhibition and conference for the space industry  
+        </p>
+        <p class="date">2021.10.27(THU)~ 30(FRI)</p>
+        <p class="place">@Findorffstraße 101 28215 Bremen, Germany</p>
+        </div>
+        </section>
+
+        <section class="main-page">
+<h2 class="main-program">Main Program</h2>
+<hr class="red-line">
+<div class="main-page-items">
+<div class="program-item"> <img class="program-img" src="./img/lechure.png" alt="Lecture"> <span class="program-name">Lecture</span> <p class="program-p">Listen to the speakers from variuos countries about the massegae of sharing and opening.</p> 
+ </div>
+ <div class="program-item"> <img class="program-img" src="./img/layer.png" alt="Exhibition"> <span class="program-name">CC Exhibition</span> <p class="program-p">Appreciate various creaion applying CC lincense of artists, organized from Art Center Nabi.</p> 
+ </div>
+ <div class="program-item"> <img class="program-img" src="./img/multiple.png" alt="Forum"> <span class="program-name">Forum</span> <p class="program-p">Have the time to share your thoughts and opinions with experts for each topic.</p> 
+ </div>
+ <div class="program-item"> <img class="program-img" src="./img/business.png" alt="Workshop"> <span class="program-name">Workshop</span> <p class="program-p">Try creating your own work using open source license rather than just watching.</p> 
+ </div>
+</div>
+
+ <button type="button" class="main-btn text-white">Join Space Tech Expo 2021</button>
+    </section>  
+
+    <section class="speakers">
+<h2 class="speaker-main">Featured Speakers</h2>
+<hr class="red-line">
+<div id="speakers" class="card-all"></div>
+    </section>
+
```

Comment:
- [ ] On the featured speakers images kindly add the background as seen in the original.

| Original | Yours |
| ------- | ----- |
| <img src="https://user-images.githubusercontent.com/33323077/133305814-b74ad833-8c3a-4b1d-89ec-d8dfab9f896e.png" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/133305882-bd4dd3e2-1be9-4353-86ef-33e216350ba9.png" height="250px"/>